src/radical/pilot/agent/lm/prte.py

```diff
@@ -56,22 +56,34 @@ def lrms_config_hook(cls, name, cfg, lrms, logger, profiler):
 
 
         # write hosts file
-        furi   = '%s/prrte.uri'   % os.getcwd()
-        fhosts = '%s/prrte.hosts' % os.getcwd()
+        furi    = '%s/prrte.uri'   % os.getcwd()
+        fhosts  = '%s/prrte.hosts' % os.getcwd()
+        vm_size = len(lrms.node_list)
 
         with open(fhosts, 'w') as fout:
             for node in lrms.node_list:
-                fout.write('%s slots=%d\n' % (node[0], lrms.cores_per_node * lrms.smt))
+                fout.write('%s slots=%d\n' % (node[0],
+                                              lrms.cores_per_node * lrms.smt))
 
         pre   = os.environ['PRRTE_PREFIX']
         prte += ' --prefix %s'     % pre
         prte += ' --report-uri %s' % furi
         prte += ' --hostfile %s'   % fhosts
-      # prte += ' --mca plm_rsh_no_tree_spawn 1'
+        prte += ' --pmca orte_state_base_verbose 1'  # prte profiling
```

Comment:
Do we want this 'always on'? Maybe enable it only when `RADICAL_PROFILE="True"`?